mach_nix/nix/NIXPKGS.json

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "https://github.com/nixos/nixpkgs-channels.git",
+  "rev": "f84aae6f67c921aa942d6b506db6509215c05591",
+  "date": "2020-08-23T11:53:32+02:00",
+  "sha256": "09zjfflwnzsb9wfk3g3khjqy29nbhdviqbbjpvjvg6nsc9p4yrbp",
+  "fetchSubmodules": false
+}
```

Comment:
Would you mind telling me again which command produces this json output?

Reply:
```
nix-shell -p nix-prefetch-git --run  "nix-prefetch-git --url https://github.com/nixos/nixpkgs-channels.git --rev refs/heads/nixos-unstable  --no-deepClone  --quiet " 
```

for me the date is worth a lot more than the ref ..

--- 
if it is ok to use json5 (with comments?)

---

Reply:
Can nix read json5? Other alternative could be toml. Toml can be parsed by nix